app/googleanalytics/ga4.py

```diff
@@ -23,14 +24,42 @@ def get_ga4():
     This route is used for pixel tracking.  It is exercised when a veteran opens an e-mail.
     The route returns a pixel image to avoid a broken icon image in notification emails.
     """
-
     # https://flask.palletsprojects.com/en/3.0.x/api/#flask.Request.args
     url_parameters_dict = request.args.to_dict()
+    current_app.logger.debug('GA4 url params: %s', url_parameters_dict)
 
     # This could raise ValidationError.
     ga4_request_validator.validate(url_parameters_dict)
 
-    current_app.logger.info(request.query_string)
+    template_name = url_parameters_dict['campaign']
+    template_id = url_parameters_dict['campaign_id']
+    name = url_parameters_dict['name']
+    source = url_parameters_dict['source']
+    medium = url_parameters_dict['medium']
+
+    content = url_parameters_dict['content'].split('/')
+    service_name = content[0]
+    service_id = content[1]
+    notification_id = content[2]
+
+    current_app.logger.info(
```

Comment:
when the method is called, we log the same information as info that we log within the method as debug? not sure I understand this?

Reply:
The difference is that the log on 29 is the raw url params, and the log on 41 is the params to the celery task, which have been processed slightly.